skyportal/handlers/api/earthquake.py

```diff
@@ -0,0 +1,616 @@
+import io
+import arrow
+from astropy.time import Time, TimeDelta
+import astropy.units as u
+import numpy as np
+from obspy.geodetics.base import gps2dist_azimuth
+from obspy.taup import TauPyModel
+from obspy.taup.helper_classes import TauModelError
+import sqlalchemy as sa
+from sqlalchemy.orm import joinedload
+from sqlalchemy.orm import sessionmaker, scoped_session
+import obspy
+
+from baselayer.app.access import auth_or_token
+from baselayer.log import make_log
+from ..base import BaseHandler
+from ...models import (
+    DBSession,
+    EarthquakeEvent,
+    EarthquakeNotice,
+    EarthquakePrediction,
+    MMADetector,
+    User,
+)
+
+log = make_log('earthquake')
+
+
+Session = scoped_session(sessionmaker(bind=DBSession.session_factory.kw["bind"]))
```

Comment:
Why not use the regular handler session?

Reply:
I am also triggering these in the background currently.